### ISSUE TITLE
Why do pixel normalization?

### ISSUE THREAD

**Issue by Chrgo** [wy1iu/sphereface#115]
A quite simple question, why normalize pixels with (img - 127.5)/128?
I didn't find any useful information. Can anyone help me? Thanks.

**Comment by wy1iu** [comment]
It is a standard data processing procedure. I think it is used not only in deep neural nets. Imagine a simple case where you just classify two points in 2D space, if you don't do such normalization, they will just lie in the positive domain, which limits the learning space of the decision boundary and incorporates unnecessary bias.

**Comment by johnnysclai** [comment]
I think @Chrgo is asking why (img - 127.5)/128 is used, instead of (img - 127.5)/127.5.

**Comment by wy1iu** [comment]
@johnnysclai I don’t think it really matters. As long as your testing and training are consistent, you will be fine.
